docs/src/pages/components/data-grid/editing/BasicEditingGrid.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { DataGrid, GridColumns, GridRowsProp } from '@material-ui/data-grid';
+import { randomCreatedDate, randomTraderName, randomUpdatedDate } from '@material-ui/x-grid-data-generator';
+
+const columns: GridColumns = [
+  { field: 'name', width: 180,  editable: true },
+  { field: 'age', type: 'number', editable: true },
+  { field: 'dateCreated', headerName:'Date Created', type: 'date', width: 180, editable: true },
+  { field: 'lastLogin',  headerName:'Last Login', type: 'dateTime', width: 220, editable: true },
+];
+
+const rows: GridRowsProp = [
+  {
+    id: 1,
+    name: randomTraderName(),
+    age: 25,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 2,
+    name:  randomTraderName(),
+    age: 36,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 3,
+    name:  randomTraderName(),
+    age: 19,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 4,
+    name:  randomTraderName(),
+    age: 28,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 5,
+    name:  randomTraderName(),
+    age: 23,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 6,
+    name:  randomTraderName(),
+    age: 10,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+];
+
+export default function BasicEditingGrid() {
+
+  return (
+    <div style={{ height: 400, width: '100%' }}>
+      <DataGrid
+        rows={rows}
+        columns={columns}
+      />
```

Comment:
One line to save vertical space?

```suggestion
      <DataGrid rows={rows} columns={columns} />
```

<img width="499" alt="Screenshot 2021-03-24 at 20 20 03" src="https://user-images.githubusercontent.com/3165635/112370979-56186600-8cde-11eb-9527-10c8f28ca22f.png">